### PR TITLE
VideoJS 5 Support

### PR DESCRIPTION
**Pull request by alex-phillips**
Changes to match the CSS stylings and API changes in the JavaScript to be compatible with VideoJS version 5.

### CODE REVIEW

**Comment by alex-phillips** [comment]
I've modified the code to be fully supported in IE8 as well as making the carousel a fully-functional sliding carousel to hold an arbitrary number of additional videos

File: videojs.relatedCarousel.js [hunk 1]
```diff
@@ -20,62 +20,198 @@
...
-        url:     ''
+        title: '',
+        url: ''
       }
     ];
 
   vjs.plugin('relatedCarousel', function(options) {
     var player = this,
-        settings = extend({}, defaults, options || {});
+      settings = extend([], defaults, options || []),
```
Comment (jgubman) [inline videojs.relatedCarousel.js]: I believe the videojs 5 way to do this is:

`settings = videojs.mergeOptions(defaults, options);`

**Comment by jgubman** [comment]
Thanks for the pull request. This adds quite a bit of complexity to the plugin past just videojs 5 support. 

I'm happy to merge these changes in, but will probably call on you if any issues arise.

**Comment by alex-phillips** [comment]
@jgubman That's no problem. We started with the version 5 forked and kind of went from there. We're using this in production on our main site, so feel free to let me know if any issues do come up. Thanks for the merge!

**Comment by jgubman** [comment]
:+1:

**Pull request merged**